docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -232,8 +232,8 @@ of the Quickstart to use the provided Docker project template.
 [[synthetics-test-suite]]
```

Comment:
In this block of text, there's a step 1, and then a step 1 and 2. Should they be continued so it's steps 1-3?